tests/unittests/algo/pbt/test_pbt.py

```diff
@@ -556,6 +565,58 @@ def test_suggest_num_population_size_fork_completed(self, space: Space, mocker):
         pbt_sample_mock.assert_called_with(2)
         pbt_fork_mock.assert_called_with(2)
 
+    def test_suggest_copy_working_dir(self, space, mocker, tmp_path):
```

Comment:
The Algo only has access to the transformed space, and so it doesn't have a way of knowing which of the trials is suggests are truly new. 

I believe there should be a test that makes sure that the algo (wrapped or not) doesn't create or copy any directories when it suggests a new trial.